apisix/cli/file.lua

```diff
@@ -65,8 +65,15 @@ local function resolve_conf_var(conf)
             local var_used = false
             -- we use '${{var}}' because '$var' and '${var}' are taken
             -- by Nginx
-            local new_val = val:gsub("%$%{%{%s*([%w_]+)%s*%}%}", function(var)
-                local v = getenv(var)
+            local new_val = val:gsub("%$%{%{%s*([%w_]+[%:%:[%w%-%.%:%?%/=]+]?)%s*%}%}", function(var)
```

Comment:
Why do we need a regex to check the default variable?

Reply:
If we don't check, It will fail to match when config contains default value.

Reply:
Sure. But we don't need to be too strict with `[%w%-%.%:%?%/=]+`? What if people want to use space in the default value?

Reply:
Done